actions/lib/action.py

```diff
@@ -53,6 +53,11 @@ def vpc_connect(self):
         del self.credentials['region']
         return boto.vpc.connect_to_region(region, **self.credentials)
 
+    def cf_connect(self):
+        region = self.credentials['region']
+        del self.credentials['region']
+        return boto.cloudformation.connect_to_region(region, **self.credentials)
```

Comment:
Thank you for your reply.
I see. It seems that is imported internally.

I think it's a good habit to import module that you use expressly for removing internal dependency.
And I recommend you to write the test for showing it's all right, when possible.

The document how to do it is prepared.
https://docs.stackstorm.com/development/pack_testing.html

Thank you.